tfhe/src/boolean/engine/bootstrapping.rs

```diff
@@ -336,7 +336,7 @@ impl Bootstrapper {
     }
 
     pub(crate) fn new_compressed_server_key(&mut self, cks: &ClientKey) -> CompressedServerKey {
-        #[cfg(not(feature = "__wasm_api"))]
+        #[cfg(not(feature = "wasm"))]
```

Comment:
switch all `feature = "__wasm_api"` for `target_arch = "wasm32"` it's something we did poorly initially as we were going fast